src/App.vue

```diff
@@ -1,16 +1,34 @@
 <template>
   <div id="app">
     <h1>{{ msg }}</h1>
+    <app-new-item @itemAdded="newItem"></app-new-item>
+    <app-item-grid :items="items" @itemDeleted="deleteItem"></app-item-grid>
   </div>
 </template>
 
 <script>
+import ItemGrid from "./components/ItemGrid.vue";
+import NewItem from "./components/NewItem.vue";
+
 export default {
   name: "app",
   data() {
     return {
-      msg: "todo list"
+      msg: "todo list",
+      items: ["Sell a car"]
     };
+  },
+  methods: {
+    newItem(item) {
+      this.items.push(item);
+    },
+    deleteItem(index) {
+      this.items.splice(index, 1);
+    }
+  },
+  components: {
+    appItemGrid: ItemGrid,
+    appNewItem: NewItem
```

Comment:
Tenhle dvojitej import mě na Vue taky trochu sere, ale údajně se to řeší.
Pokud máš WEbstorm/Phpstorm pomůžu s konfigurací autoimportu. 
To přejmenování v components už nemusíš dělat.
Takhle vypadá kód z produkce
![obrazek](https://user-images.githubusercontent.com/15363559/84013873-cd605180-a979-11ea-9e12-800bfe8ea7df.png)